### ISSUE TITLE
Uncaught (in promise) TypeError: Cannot read property 'contentWindow' of null

### ISSUE THREAD

**Issue by diantekyrie** [ampproject/amphtml#23989]
##What's the issue?
Error with Amp-video-iframe

Briefly describe the bug.
Cannot read property 'contentWindow' of null. 

## How do we reproduce the issue?
1. Opt into Canary/RC
2. Open javascript console
3. Go to page https://www.google.com/amp/s/www.cnet.com/google-amp/news/iphone-11-pro-11r-max-new-specs-features-price-expect-apple-every-rumor-2019-pro/
4. Hit back button on browser

## What browsers are affected?
macOSX Chrome Version: 76.0.3809.100

## Which AMP version is affected?
Canary/RC: 1908150025400

Video: https://drive.google.com/open?id=1jCBkHV9Qk7Ysirb3a5y95YmPSAjV1zCK

**Comment by choumx** [comment]
This is because the iframe's `contentWindow` is cleaned up by the time the promise chain is invoked.
